composer.json

```diff
@@ -6,7 +6,11 @@
     "type": "project",
     "require": {
         "php": ">=5.5.9",
-        "laravel/framework": "5.2.*"
+        "laravel/framework": "5.2.*",
+        "doctrine/dbal": "~2.3",
+        "twilio/sdk": "5.0.0-RC19",
```

Comment:
Please upgrade Twilio dependency to `"twilio/sdk": "^5.0",` stable release is out. Lets use this one from now on